vbfml/scripts/err_analysis

```diff
@@ -403,7 +403,7 @@ def evaluate(ctx, input_dir: str, sequence_type: str) -> None:
     "--save_name",
     required=False,
     type=str,
-    default=None,
+    default= "",
```

Comment:
Thanks for the updates @pmayenco, I was just looking at this line. Should we give this parameter a non-empty default, so that in case it is not specified, the name of the output files/directory still makes sense? Maybe something like `default="history"` so that by default, output dir is named `comparison_history`. Right now it looks like it will be saved as `comparison_` (similar for the PDF files).

Reply:
done